tensorflow/compiler/xla/service/cpu/cpu_compiler.cc

```diff
@@ -648,6 +649,15 @@ Status CpuCompiler::RunHloPassesThroughLayoutAssn(
   pipeline.AddPass<CallInliner>(/*single_call_site=*/true);
   pipeline.AddPass<BatchDotSimplification>();
   pipeline.AddPass<DotDecomposer>();
+
+  // Rewrite to custom calls with target as oneDNN library calls.
+#if defined(INTEL_MKL) && defined(ENABLE_ONEDNN_V3)
+  // AOT compiled code runs in single thread.
```

Comment:
can you add more detail here, e.g. no thread pool is available to execute oneDNN primitive.